example/tests/test_models.py

```diff
@@ -55,15 +68,12 @@ def test_template_check_should_detect_multiple_variable_formats(self):
 
     @override_settings(PYNOTIFY_TEMPLATE_TRANSLATE=True)
     def test_template_should_be_translated(self):
-        activate('cs')
+        with override('cs'):
```

Comment:
Nothing related to this PR, I'm just switching to context processor, which is better approach.